src/components/atom/AppBackButton.tsx

```diff
@@ -1,13 +1,14 @@
-import {Link} from "react-router-dom";
+import {useNavigate} from "react-router-dom";
 import LeftArrowIcon from "../icons/LeftArrowIcon";
 
+const AppBackButton = () => {
+    const navigate = useNavigate()
 
-const AppBackButton = ({link}: { link: string }) => {
     return (
-        <Link to={link} className="flex h-12 w-12 items-center justify-center rounded-full bg-white shadow-md">
+        <button onClick={() => navigate(-1)} className="flex h-12 w-12 items-center justify-center rounded-full bg-white shadow-md">
```

Comment:
@melinahernandeztheam The `-1` is never going to change to another value so I think it's OK to leave it like that.

Reply:
I didn't say that because of duplication or change, it is because of good practices and semantics. We should never leave a number on its own or a string literal, it is much easier to read `navigate(AppRoute.BACK) ` along with other beneficts, easier to search, easier to refactor, no duplication

Reply:
@melinahernandeztheam In general that's true. However, in some situations it makes sense to leave a number or string literal as-is without wrapping it in a constant or variable. In this case `-1` is more concise and simpler than using an enum.

Let's agree to disagree, it's not a bid deal. I'll add the route 👍🏻